core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -393,11 +388,16 @@ class TransactionStateManager(brokerId: Int,
       loadingPartitions.add(partitionAndLeaderEpoch)
     }
 
-    def loadTransactions(): Unit = {
+    def loadTransactions(startTimeMs: java.lang.Long): Unit = {
+      val schedulerTime = time.milliseconds() - startTimeMs
       info(s"Loading transaction metadata from $topicPartition at epoch $coordinatorEpoch")
       validateTransactionTopicPartitionCountIsStable()
 
       val loadedTransactions = loadTransactionMetadata(topicPartition, coordinatorEpoch)
+      val endTimeMs = time.milliseconds()
+      val timeLapse = endTimeMs - startTimeMs
```

Comment:
nit: maybe we call this `totalLoadingTimeMs` for consistency

Reply:
Done.